wiki/it/it-Getting-Started.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: wiki
+title: "Primi Passi"
+lang: "it"
+permalink: "/wiki/Getting-Started"
+---
+# Primi Passi
+## Avrai bisogno di
+
+1. Un PC con **Windows 10**, oppure con **MacOS X El Capitan (v10.11) o versione superiore**, o **Linux**. Requisiti mini sono una CPU da 1,5GHZ.
+1. **Una connessione Ethernet via cavo e non WI-FI**(su connessione WI-FI si posso verificare difficolta di utilizzo e problemi di ascolto). Molte delle attuali linee sono sufficienti per garantire il buon funzionamento dell'applicazione([clicca qui per maggiori dettagli](Quality,-delay-and-network-bandwidth)). Si ricorda che una connessione con 10Mbits in down e 1Mbits in up non è sufficiente per garantire un buon funzionamento e non avere problemi di ascolto.
```

Comment:
Thank you very much for your translation!
Quality,-delay-and-network-bandwidth has been renamed to Network-Requirements. Please rename it and have a look at https://github.com/jamulussoftware/jamulussoftware.github.io/blob/main/wiki/en/en-Getting-Started.md since we now open it in a new tab.